packages/ssr/Cargo.toml

```diff
@@ -29,3 +29,6 @@ argh = "0.1.4"
 serde = "1.0.120"
 serde_json = "1.0.61"
 fs_extra = "1.2.0"
+
+[features]
+incremental = ["dep:tokio"]
```

Comment:
can we enable the feature by default so that this is non-breaking?
```
default = ["incremental"]
```

Reply:
Absolutely.